packages/web/src/scss/tools/_tokens.scss

```diff
@@ -21,3 +21,43 @@
         @return $fallback-value;
     }
 }
+
+// Function to generate a map of responsive values for a given property, size, and token prefix.
+//
+// Example:
+// generate-responsive-map('border-radius', 'radius', 'small', 'form-field', 4px) might return:
+// (
+//   'border-radius-mobile': 4px,
+//   'border-radius-tablet': 6px,
+//   'border-radius-desktop': 8px,
+// )
+//
+// Parameters are:
+// * $property: the base CSS property name (e.g. 'border-radius')
+// * $size: the size variant (e.g. 'small', 'medium')
+// * $variable-prefix: prefix used for the token variable names (e.g. 'form-field')
+// * $token-name: token name (e.g. 'radius')
+// * $fallback: value to use if token is not found
+@function generate-responsive-map($args) {
+    $result: ();
+    $property: map.get($args, 'property');
+    $size: map.get($args, 'size');
+    $variable-prefix: map.get($args, 'variable-prefix');
+    $token-name: map.get($args, 'token-name');
+    $fallback: map.get($args, 'fallback');
```

Comment:
Well, a nice idea 🙂, but this is not what @crishpeen meant. The arguments can be listed inside of the function brackets. What we need is use argument names in function calls:

```scss
generate-responsive-map(
    $property: 'border-radius',
    $token-name: 'radius',
    $size: 'small',
    $variable-prefix: 'form-field',
    $fallback: 4px
);
```